deployment/layoutCmd.py

```diff
@@ -0,0 +1,84 @@
+import os
+from kubernetes import client, config
+import yaml
+
+
+def generate_layout(output_file):
```

Comment:
Put it to a single module?

Reply:
What's the benefit of module? Here we don't need it.

Reply:
Just a suggestion.